packages/playground/src/components/dedicated_nodes_table.vue

```diff
@@ -0,0 +1,281 @@
+  { title: "GPU", key: "num_gpu", sortable: false },
+  {
+    title: "Price (USD)",
+    key: "price",
+    sortable: false,
+  },
+  {
+    title: "Actions",
+    key: "actions",
+    sortable: false,
+  },
+];
+const profileManager = useProfileManager();
+const pageSize = ref(10);
+const page = ref(1);
+const expanded = ref([]);
+const tabs = [{ label: "Rentable" }, { label: "Rented" }, { label: "Mine" }];
+const activeTab = ref(0);
+const loading = ref(false);
+const nodes = ref<any[]>();
+const nodesCount = ref(0);
+const filterInputs = ref<DedicatedNodeFilters>(DedicatedNodeInitializer);
+const isValidForm = ref<boolean>(false);
+const isFormLoading = ref<boolean>(true);
+
+const tabParams = {
+  0: {
+    rentable: true,
+    status: NodeStatus.Up,
+    retCount: true,
+  },
+  1: {
+    rented: true,
+    status: NodeStatus.Up,
+    retCount: true,
+  },
+  2: {
+    rented: true,
+    rentedBy: profileManager.profile?.twinId,
+    retCount: true,
+  },
+};
+
+onMounted(async () => {
+  await loadData();
+});
+
+const _loadData = async () => {
+  const params = tabParams[activeTab.value as keyof typeof tabParams];
+
+  if (!params) {
+    return;
+  }
+
+  loading.value = true;
+  isFormLoading.value = true;
+  try {
+    const data = await gridProxyClient.nodes.list({
+      ...params,
+      size: pageSize.value,
+      page: page.value,
+      totalSru: filterInputs.value.total_sru.value ? toBytes(+filterInputs.value.total_sru.value) : undefined,
+      totalMru: filterInputs.value.total_mru.value ? toBytes(+filterInputs.value.total_mru.value) : undefined,
+      totalHru: filterInputs.value.total_hru.value ? toBytes(+filterInputs.value.total_hru.value) : undefined,
+      totalCru: filterInputs.value.total_cru.value ? +filterInputs.value.total_cru.value : undefined,
+      gpuVendorName: filterInputs.value.gpu_vendor_name.value ? filterInputs.value.gpu_vendor_name.value : "",
+      gpuDeviceName: filterInputs.value.gpu_device_name.value ? filterInputs.value.gpu_device_name.value : "",
+    });
+
+    if (data.count === 0) {
+      loading.value = false;
+      nodes.value = [];
+      nodesCount.value = 0;
+      isFormLoading.value = false;
+      return;
+    }
+
+    const grid = await getGrid(profileManager.profile!);
+
+    const pricePromises = data.data.map(async item => {
+      const fee = await grid?.contracts.getDedicatedNodeExtraFee({ nodeId: item.nodeId });
+      const price = await grid?.calculator.calculateWithMyBalance({
+        cru: item.total_resources.cru,
+        hru: toGigaBytes(item.total_resources.hru.toString()),
+        mru: toGigaBytes(item.total_resources.mru.toString()),
+        sru: toGigaBytes(item.total_resources.sru.toString()),
+        ipv4u: false,
+        certified: item.certificationType === CertificationType.Certified,
+      });
+      return {
+        ...item,
+        price: (price?.dedicatedPrice ?? 0 + (fee || 0)).toFixed(3),
+        discount: price?.sharedPackage.discount ?? 0,
```

Comment:
Property 'discount' does not exist on type 'string'

Reply:
- both of type number, where is type `string` ??

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/4d49675d-a228-4192-8575-050e3e860c58)

Reply:
Forget about it, it was my **VSCODE :D